core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -145,6 +149,8 @@ public void start(Boolean isPrimary) throws IOException {
         }
       }
       mAsyncJournalWriter = new AsyncJournalWriter(mJournalWriter);
+      mAsyncAuditLogWriter = new AsyncUserAccessAuditLogWriter();
+      mAsyncAuditLogWriter.start();
```

Comment:
The alluxio master process runs multiple masters for different purposes, e.g. FileSystemMaster, BlockMaster, KeyValueMaster, etc. All of these masters extend `AbstractMaster`.